Sources/OTel/OTelCore/OTel+Configuration+DiagnosticLogger.swift

```diff
@@ -52,3 +52,24 @@ extension Logger {
         return result
     }
 }
+
+extension Logger {
+    func log(
```

Comment:
I'd also love to see this extension in swift-log proper (not a blocker for this PR).

Reply:
Interesting idea. I don't know if it's generalisable enough as people may have opinions on how they log the level.